### PR TITLE
Add opencontainers labels to Dockerfile (#410)

### PR DESCRIPTION
**Pull request by mstoetzer**
According to https://github.com/opencontainers/image-spec/blob/main/annotations.md\#pre-defined-annotation-keys

This PR fixes issue #410 

## Checklist
* [x] I have signed the CLA
* [x] I have updated/added any relevant documentation

### What's the goal of this PR?
Provide opencontainers labels in the container image

### What changes did you make?
Added opencontainers labels to the Dockerfile
See: https://github.com/opencontainers/image-spec/blob/main/annotations.md#pre-defined-annotation-keys

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,14 @@
 FROM alpine:3.15.0
 
+LABEL org.opencontainers.image.authors="@sudermanjr @lucasreed" \
```
Comment (sudermanjr) [inline Dockerfile:3]:
```suggestion
LABEL org.opencontainers.image.authors="FairwindsOps, Inc." \
```

File: Dockerfile [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+      org.opencontainers.image.vendor="FairwindsOps"
```
Comment (sudermanjr) [inline Dockerfile:4]:
```suggestion
      org.opencontainers.image.vendor="FairwindsOps, Inc."
```

**Review by sudermanjr — CHANGES_REQUESTED** [review]
Thanks!

I made a couple suggestions on naming for the authors and vendor fields.

**Comment by sudermanjr** [comment]
Also, it looks like the container build is failing:

```
error: failed to solve: rpc error: code = Unknown desc = failed to solve with frontend dockerfile.v0: failed to create LLB definition: dockerfile parse error line 5: unknown instruction: ORG.OPENCONTAINERS.IMAGE.TITLE="GOLDILOCKS"
```

**Comment by mstoetzer** [comment]
Closed this by accident, please follow this PR #413

**Pull request closed without merging**